project_app/styles/AdminLayout.module.css

```diff
@@ -0,0 +1,19 @@
+.adminLayout {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+  background-color: #353535;
+}
+
+.adminContent {
+  flex-grow: 1;
+  width: 100%;
+  display: flex;
+  justify-content: center; /* centers content horizontally */
+  align-items: center; /* centers content vertically */
+}
+
```

Comment:
we use styled components in this project.
pls contact @twentymurial33 if you need help

Reply:
@enricodelarosa we have the `AdminLayout` component now using Styled Components 👍